tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DatabaseMonitoringPropagator.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="DatabaseMonitoringPropagator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet
```

Comment:
You should add unit tests. I know you already have ITests but you can maybe tests more things in UTests, and more importantly, it's easier to run them and thus to have a quick feedback on your implementation